runtime/frequency/src/lib.rs

```diff
@@ -467,6 +468,7 @@ pub type MaxReleaseSchedules = ConstU32<{ MAX_RELEASE_SCHEDULES }>;
 // the descriptions of these configs.
 impl pallet_time_release::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
+	type Balance = Balance;
```

Comment:
Here we update `type Balance` because `time-release` adds the `MaybeSerializeDeserialize` trait.